e2e/app/component/workspace-card.ts

```diff
@@ -176,15 +176,10 @@ export default class WorkspaceCard extends CardBase {
     });
     const element = BaseElement.asBaseElement(this.page, elementHandle);
     const name = await getPropValue<string>(elementHandle, 'textContent');
+    await element.click();
     if (waitForDataPage) {
-      const navPromise = this.page.waitForNavigation({ waitUntil: ['load', 'networkidle0'] });
-      await element.click();
-      await navPromise;
```

Comment:
Failed CircleCI [run](https://app.circleci.com/pipelines/github/all-of-us/workbench/16763/workflows/909d1f27-6de8-440a-829a-33fd6e5a0193/jobs/187721/tests): `waitForNavigation` is flaky. It doesn't work on rare occasions: 
```
failure: [
  'TimeoutError: Navigation timeout of 90000 ms exceeded\n' +
    '    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/LifecycleWatcher.js:106:111\n' +
    '    at FrameManager.waitForFrameNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:127:23)\n' +
    '    at Frame.waitForNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:441:16)\n' +
    '    at Page.waitForNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:794:16)\n' +
    '    at WorkspaceCard.clickWorkspaceName (/home/circleci/workbench/e2e/app/component/workspace-card.ts:182:7)\n' +
    '    at Object.findOrCreateWorkspace (/home/circleci/workbench/e2e/utils/test-utils.ts:183:7)\n' +
    '    at /home/circleci/workbench/e2e/tests/datasets/dataset-snowman-menu-actions.spec.ts:119:5'
]
```